packages/headless/src/state/product-listing-app-state.ts

```diff
@@ -10,13 +10,13 @@ import {
   FacetSection,
```

Comment:
I believe you can delete this file completely. There should be nothing importing the `ProductListingAppState` type once we remove the product listing sub-package.

Reply:
@fbeaudoincoveo this is actually used in packages/headless/src/api/commerce/product-listings/product-listing-api-client.ts

Reply:
I think we can modify the type `AsyncThunkProductListingOptions` to not extends this without any drawback. And then we could delete the file.